ui/app/adapters/generated-item-list.js

```diff
@@ -7,20 +7,22 @@ import ApplicationAdapter from './application';
 import { task } from 'ember-concurrency';
 import { service } from '@ember/service';
 
-export default ApplicationAdapter.extend({
-  store: service(),
-  namespace: 'v1',
-  urlForItem() {},
-  dynamicApiPath: '',
+export default class GeneratedItemAdapter extends ApplicationAdapter {
+  @service store;
+  namespace = 'v1';
+  urlForItem() {}
+  dynamicApiPath = '';
```

Comment:
I was going to ask if this should be tracked, but I don't actually see it being used anywhere?  It's not passed in where `this.urlForItem()` is invoked in the `path-help` service on line 225
```js
urlForQueryRecord(id, modelName) {
 return this.urlForItem(id, modelName);
}